wolox-books/src/app/screens/books/book-list/book-list.component.html

```diff
@@ -1,2 +1,11 @@
-<p>book-list works!</p>
-<a routerLink="/app/detail">Detail</a>
+<div class="container-list content">
+  <div id="book-list" class="book-list">
+    <a *ngFor="let book of bookList$ | async" routerLink="/app/detail">
+      <div class="book">
```

Comment:
This div is not necessary, you can apply the styles to the `a` tag

Reply:
Done